src/sys/socket/addr.rs

```diff
@@ -973,4 +1082,37 @@ pub mod sys_control {
             write!(f, "id: {} unit: {}", self.id(), self.unit())
         }
     }
+
+    impl fmt::Debug for SysControlAddr {
+        fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+            fmt::Display::fmt(self, f)
+        }
+    }
+}
+
+#[derive(Clone, Copy, Eq, Hash, PartialEq)]
+pub struct EtherAddr(pub u8, pub u8, pub u8, pub u8, pub u8, pub u8);
```

Comment:
This should be a newtype around either `sockaddr_dl` or `sockaddr_ll` depending on the platform.

Reply:
so ?

Reply:
So what?